features/filiatie.feature

```diff
@@ -0,0 +1,618 @@
+    | splitsen van perceel in nieuwe percelen   | vervallen                                    | ja                                                  |
+    | samenvoegen van percelen                  | vervallen                                    | ja                                                  |
+    | splitsen perceel in appartementsrechten   | actueel                                      | ja                                                  |
+    | ondersplitsing van appartementsrecht      | sluimerend                                   | ja                                                  |
+    | wijzigingssplitsing van appartementsrecht | vervallen                                    | nee                                                 |
+
+
+    Naast de filiatie worden relaties naar de grondpercelen waaruit het appartementsrecht is ontstaan of naar de appartementsrechten horend bij het perceel geleverd, zodat direct de bijbehorende grondpercelen dan wel bijbehorende appartementsrechten kunnen worden opgehaald, zonder eerst via eventuele sluimerende appartementsrechten de filiatie af te hoeven lopen.
+
+    #TODO: invullen onroerende zaak identificaties
+
+    Rule: Na splitsing van een perceel wordt filiatie opgenomen bij het vervallen perceel en bij de nieuwe percelen
+        Scenario: Opvragen van een vervallen perceel na splitsing
+            Gegeven Perceel met identificatie "P1" is gesplitst naar percelen "P2" en "P3"
+            En Perceel met identificatie "P1" is niet ontstaan uit splitsing of vereniging van een ander perceel
+            En Perceel met identificatie "P2" is niet verder gesplitst of verenigd
+            En Perceel met identificatie "P3" is niet verder gesplitst of verenigd
+            Als "/kadastraalonroerendezaken/P1" wordt gevraagd
+            Dan bevat het antwoord "isOvergegaanIn" met waarde:    
+                """
+                [
+                    {
+                        "aard": {
+                            "code": "20",
+                            "waarde": "Splitsen perceel"
+                        },
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P2"
+                        }
+                    },
+                    {
+                        "aard": {
+                            "code": "20",
+                            "waarde": "Splitsen perceel"
+                        },
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P3"
+                        }
+                    }
+                ]
+                """
+            En bevat het antwoord "indicatieVervallen": true
+            En bevat het antwoord property "einddatum" met een waarde
+            En is de kadastraal onroerende zaak niet sluimerend
+            En bevat het antwoord geen property "bijbehorendeGrondperceelIdentificaties"
```

Comment:
ik kan het niet valideren omdat mijn apikey het niet doet, maar was het niet zo dat we bij lege collectie een lege collectie terug krijgen en niet null?

Reply:
dit gaat niet over een collectie van resources, maar over een property met een array van identificaties. Als een array property geen enkel item bevat, wordt het property niet opgenomen in het antwoord.